ui/components/src/utils/theme-gen.ts

```diff
@@ -39,7 +39,21 @@ export function generateChartsTheme(muiTheme: MuiTheme, echartsTheme: EChartsThe
       left: 20,
       containLabel: true,
     },
-    color: ['#8dd3c7', '#fb8072', '#80b1d3', '#bebada', '#fdb462'],
+    // Accessible categorical palette from: https://colorbrewer2.org/#type=qualitative&scheme=Paired&n=12
```

Comment:
It's worth noting that they don't list that value as colorblind safe. If you check this checkbox, they'll filter it down.

<img width="166" alt="image" src="https://user-images.githubusercontent.com/396962/233154342-79346e45-640f-4cc2-a814-2d09efadc6de.png">

It looks like the most the can do safely is:

11 diverging
<img width="268" alt="image" src="https://user-images.githubusercontent.com/396962/233154678-f7191680-86e5-4a8c-baa8-50aea90301e1.png">

9 sequential
<img width="268" alt="image" src="https://user-images.githubusercontent.com/396962/233154854-93b3a96d-35e1-4931-9276-3657e5c3df15.png">

4 categorical
<img width="268" alt="image" src="https://user-images.githubusercontent.com/396962/233154952-37a9832c-e269-4fcb-8f2f-db4a9f26562b.png">

Reply:
This person has some recommendations that might also be useful.

https://personal.sron.nl/~pault/#sec:qualitative

Reply:
Oh wow, yeah I missed that checkbox! So the only option for colorblind safe has four colors 😬

I really worry that repeating on the 5th color is a dealbreaker with the amount of data that we need to be able to show (so many queries are going to return series where many lines are exactly the same color), to me colorblind safe is less about every single color having the maximum amount of contrast and more that certain hues are favored since certain hues are easier for me to distinguish (people with other types of colorblindness could potentially opt-in to a different palette or maybe we give some control over generating hues in panel options later). The whole errors === 'red' hues has a great side effect where those hues are off limits to other metrics, meaning the 'Auto' palette favors the cooler side of the ROYGBIV spectrum. 

The other piece to emphasize is we have _complete_ control to adjust this based on feedback (the [color-hash library](https://github.com/zenozeng/color-hash#custom-hue) is so powerful and seems like a core architectural piece in Perses that we can keep iterating on).